IHP/ControllerSupport.hs

```diff
@@ -169,6 +172,40 @@ getHeader :: (?context :: ControllerContext) => ByteString -> Maybe ByteString
 getHeader name = lookup (Data.CaseInsensitive.mk name) (Network.Wai.requestHeaders request)
 {-# INLINABLE getHeader #-}
 
+-- | Set a header value for a given header name.
+--
+-- >>> setHeader ("Content-Language", "en")
+--
+setHeader :: (?context :: ControllerContext) => Header -> IO ()
+setHeader header = do
+    maybeHeaders <- Context.maybeFromContext @[Header]
+    let headers = fromMaybe [] maybeHeaders
+    Context.putContext (header : headers)
+{-# INLINABLE setHeader #-}
+
+-- | Add headers to current response
+-- | Returns a Response with headers
+--
+-- >>>  addResponseHeaders [("Content-Type", "text/html")] response
```

Comment:
```suggestion
-- > addResponseHeaders [("Content-Type", "text/html")] response
```

The `>>>` haddock syntax always expects a result object in the next line. As it's not here we need to use a `>`